.github/workflows/verify-ensurepip-wheels.yml

```diff
@@ -1,4 +1,4 @@
-name: Verify bundled pip and setuptools
+name: Verify bundled pip
```

Comment:
For this file I think it would be fine to not be specific in the names, like:
Verify bundled wheels
Compare checksums of bundled wheels to the ones on PyPI